.github/workflows/regression.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: regression testing
+
+# Prevent keeping resources busy when a branch/PR is updated
+# https://docs.github.com/en/actions/using-jobs/using-concurrency
+concurrency:
+  group: ${{ github.repository }}-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  regression:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { tool: klayout, part: drc, test: main }
+          - { tool: klayout, part: drc, test: switch }
+          - { tool: klayout, part: lvs, test: main }
+          - { tool: klayout, part: lvs, test: switch }
+
+    name: ${{ matrix.part }} | ${{ matrix.test }}
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Testing ${{ matrix.part }} for ${{ matrix.test }}
+        run: >
```

Comment:
Looks like this currently fail:
https://github.com/efabless/globalfoundries-pdk-libs-gf180mcu_fd_pv/actions/runs/3949971157/jobs/6761875635

Should `>` be `|` ?

Reply:
@proppy Could you please resolve this?